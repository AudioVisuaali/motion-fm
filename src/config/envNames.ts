export const API_PORT = "API_PORT";
export const LOGGING_LEVEL = "LOGGING_LEVEL";

export const HUE_ADDRESS = "HUE_ADDRESS";
export const HUE_USERNAME = "HUE_USERNAME";
export const HUE_SENSOR_IDS = "HUE_SENSOR_IDS";