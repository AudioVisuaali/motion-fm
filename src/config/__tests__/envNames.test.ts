import * as envNames from "../envNames";

describe("config", () => {
  describe("envNames", () => {
    it("should have the correct values", () => {
      expect(envNames).toMatchSnapshot();
    });
  });
});
