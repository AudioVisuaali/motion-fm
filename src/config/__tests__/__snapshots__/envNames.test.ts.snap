// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config envNames should have the correct values 1`] = `
{
  "API_PORT": "API_PORT",
  "HUE_ADDRESS": "HUE_ADDRESS",
  "HUE_SENSOR_IDS": "HUE_SENSOR_IDS",
  "HUE_USERNAME": "HUE_USERNAME",
  "LOGGING_LEVEL": "LOGGING_LEVEL",
  "TZ": "TZ",
}
`;
